quickwit/quickwit-janitor/src/metrics.rs

```diff
@@ -34,6 +43,37 @@ impl Default for JanitorMetrics {
                 &[],
                 ["index"],
             ),
+            gc_deleted_splits: new_counter(
+                "gc_deleted_splits_count",
+                "Total number of splits deleted by the garbage collector.",
+                "quickwit_janitor",
+                &[],
+            ),
+            gc_deleted_bytes: new_counter(
+                "gc_deleted_bytes_count",
+                "Total number of bytes deleted by the garbage collector.",
+                "quickwit_janitor",
+                &[],
+            ),
+            gc_failed_deleted_splits: new_counter(
+                "gc_deleted_splits_error_count",
```

Comment:
```suggestion
            gc_delete_splits_failure: new_counter(
                "gc_delete_splits_failures_total",
```

Reply:
Personally I would have used a label "result" with two values.
"success" and "failure".